editor/inspector/editor_inspector.cpp

```diff
@@ -1077,15 +1085,22 @@ void EditorProperty::gui_input(const Ref<InputEvent> &p_event) {
 
 	Ref<InputEventMouseButton> mb = p_event;
 
-	if (mb.is_valid() && mb->is_pressed() && mb->get_button_index() == MouseButton::LEFT) {
+	if (mb.is_valid() && mb->get_button_index() == MouseButton::LEFT) {
+		bool is_button_down = mb->is_pressed();
+
 		Vector2 mpos = mb->get_position();
 		if (is_layout_rtl()) {
 			mpos.x = get_size().x - mpos.x;
 		}
 
-		select();
+		if (is_button_down) {
+			select();
+		}
 
-		if (keying_rect.has_point(mpos)) {
+		if (keying_rect.has_point(mpos) && is_button_down) {
```

Comment:
```suggestion
		if (is_button_down && keying_rect.has_point(mpos)) {
```
For all these, cheaper condition first